### PR TITLE
Update request

### PR DESCRIPTION
**Pull request by 0xSinTax**

### CODE REVIEW

File: streamers.csv [hunk 1]
```diff
@@ -129,6 +129,7 @@ mttaggart,https://www.youtube.com/taggarttech
 MVLWVR3,
 NahamSec,https://www.youtube.com/c/Nahamsec
 Nirloy,
+OALabsLive, https://www.youtube.com/oalabs
```
Comment (XeSquirmy) [inline streamers.csv:132]: There should be no space after the comma

**Review by XeSquirmy — CHANGES_REQUESTED** [review]
one thing then you're good

File: streamers.csv (hunk 1 above)
Comment (0xSinTax) [reply streamers.csv]: Done.

**Review by goproslowyo — APPROVED** [review]
Looks good, thanks.

**Comment by goproslowyo** [comment]
Merged!

**Pull request merged**